cpp/models/abm/infection.cpp

```diff
@@ -21,17 +21,45 @@
 #include "abm/infection.h"
 #include <math.h>
 
-TimePoint mio::abm::ViralLoad::determine_end_date(TimePoint start_date)
+namespace mio
+{
+namespace abm
+{
+
+void ViralLoad::draw_viral_load()
+{
+    // These numbers are subject to change, They are going to be based on distributions backed from data.
+    m_peak    = 5.0;
+    m_incline = 1.0;
+    m_decline = -1.0;
+}
+
+TimePoint ViralLoad::determine_end_date(const TimePoint& start_date)
 {
     return start_date + TimeSpan(int(m_peak / m_incline - m_peak / m_decline));
 }
 
-double mio::abm::Infection::get_infectivity(const TimePoint t) const
+double ViralLoad::get_viral_load(const TimePoint& t, const TimePoint& start_date) const
 {
-    return 1 / (1 + exp(-(m_log_norm_alpha + m_log_norm_beta * get_viral_load(t))));
+    if (t.seconds() <= start_date.seconds() + m_peak / m_incline) {
```

Comment:
m_peak / m_incline is not in seconds, is it? why can they be compared like this?

Reply:
Let's say m_peak is in the unit U, then m_incline will be in U/second, so it should be in seconds. Otherwise, one can always easily redefine the time unit scale to days (or anything else) if that is more convenient. Since we don't have the data right now, we put some placeholder and seconds is a common unit size for time.